openshift/template.yaml

```diff
@@ -0,0 +1,125 @@
+apiVersion: v1
+kind: Template
+labels:
+  template: 3scale-admin-gateway
+metadata:
+  name: 3scale-admin-gateway
+  annotations:
+    description: 3scale-admin-gateway
+objects:
+- apiVersion: v1
+  kind: DeploymentConfig
+  metadata:
+    labels:
+      service: 3scale-admin-gateway
+    name: 3scale-admin-gateway
+  spec:
+    selector:
+      service: 3scale-admin-gateway
+    template:
+      metadata:
+        labels:
+          service: 3scale-admin-gateway
+      spec:
+        containers:
+        - env:
+          - name: F8A_DEBUG
```

Comment:
Not used-not needed?

Reply:
I don't understand this question, could you please explain?

Reply:
If this `F8A_DEBUG` is not used by the container, then you can remove it from the template. I guess it's just copy+paste leftover.